README.md

```diff
@@ -1805,7 +1805,9 @@ IEEE Access, 2020.
 - <a name="Banerjee20arxiv"></a>Banerjee, S., Wang, Z. W., Chopp, H. H., Cossairt, O., Katsaggelos, A.,  
 *[Quadtree Driven Lossy Event Compression](https://arxiv.org/pdf/2005.00974.pdf)*,  
 arXiv 2020. [PDF](https://arxiv.org/pdf/2005.00974.pdf)
-
+- <a name="Khan21IoT-J"></a>N. Khan, K. Iqbal and M. G. Martini,  
+*[Time-Aggregation-Based Lossless Video Encoding for Neuromorphic Vision Sensor Data](https://ieeexplore.ieee.org/document/9136663)*,  
```

Comment:
We prefer to replace the url link with the DOI: https://doi.org/10.1109/JIOT.2020.3007866

Reply:
I've changed the url link